.devcontainer/devcontainer.json

```diff
@@ -1,36 +1,67 @@
 {
 	"name": "Gadgetron",
-
 	"build": {
 		"dockerfile": "Dockerfile",
 		"args": {
-			"BASE_IMAGE": "ghcr.io/gadgetron/gadgetron/gadgetron_ubuntu_2004_cuda11_cudnn8_base"
+			"BASE_IMAGE": "ghcr.io/gadgetron/gadgetron/gadgetron_ubuntu_dev_cuda:latest"
```

Comment:
This appears to be a private repo
``` Step 1/21 : ARG BASE_IMAGE=ghcr.io/gadgetron/gadgetron/gadgetron_ubuntu_2004_cuda11_cudnn8_base
Step 2/21 : FROM ${BASE_IMAGE}
[2022-01-11T14:08:08.748Z] Head "https://ghcr.io/v2/gadgetron/gadgetron/gadgetron_ubuntu_dev_cuda/manifests/latest": unauthorized
```

Reply:
It is until we merge this PR and start pushing these images. Until then, you can build it locally with:

```
./build-images --type dev --flavor cuda
```